modules/javafx.graphics/src/main/java/com/sun/javafx/scene/DirectionalLightHelper.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2021

Reply:
Can the script do all these corrections?

Reply:
For existing files, yes. For this file (along with `NGDirectionalLight` and `DirectionalLight`), we will end up with `2020, 2021,` after running the script. Not a big deal, but since you have to make other changes anyway to fix the typos you might as well fix this, too.